### PR TITLE
Update icons to use $ icon

### PR DESCRIPTION
**Pull request by AndrewGable**
Please review @puneetlath cc @Expensify/design 

### Details
Updates the icon asset for all clients:

1. sign in page
2. chat switcher
3. app icon
4. favicon
5. Push notification asset on Android
6. README.md

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/148907

### Tests
1. Run `npm run ios` -> Verify icon is updated
2. Run `npm run android` -> Verify icon is updated
3. Run `npm run web` -> Verify icon is updated
4. Run `npm run desktop-build` -> verify icons are updated

##### Desktop
Screen Shot 2020-12-22 at 3 37 02 PM
Screen Shot 2020-12-22 at 3 37 13 PM
Screen Shot 2020-12-22 at 3 37 30 PM
Screen Shot 2020-12-22 at 3 37 35 PM

##### Android
Screen Shot 2020-12-22 at 3 17 32 PM
Screen Shot 2020-12-22 at 3 17 17 PM
Screen Shot 2020-12-22 at 2 58 11 PM

##### iOS
Screen Shot 2020-12-22 at 2 44 46 PM

##### Web
Screen Shot 2020-12-22 at 2 42 38 PM
Screen Shot 2020-12-22 at 2 42 22 PM
Screen Shot 2020-12-22 at 2 28 36 PM
Screen Shot 2020-12-22 at 2 28 29 PM

### CODE REVIEW

**Comment by puneetlath** [comment]
Web favicon, chat switcher, login page icons all look good.

**Comment by megankelso** [comment]
cc @Expensify/design but looks good to me!

**Comment by shawnborton** [comment]
Yup looks good to me too. Just checking that the web favicon when you have unreads was updated too?

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 <div align="center">
-  <img src="https://use.expensify.com/assets/logo/download/expensify-app-logo_circular/expensify-app-logo_circular.png" width="64" height="64">
+    <a href="https://Expensify.cash">
+        <img src="https://raw.githubusercontent.com/Expensify/Expensify.cash/master/web/favicon.png" width="64" height="64" alt="Expensify.cash Icon">
```
Comment (marcaaron) [inline README.md:3]: NAB not sure but can/should we be using a relative path for this?

**Review by marcaaron — COMMENTED** [review]
Changes look good just testing now and have NAB comment.

**Comment by AndrewGable** [comment]
@shawnborton - Yes it updates, but, I am not sure it looks great:

Screen Shot 2020-12-22 at 4 37 40 PM

**Comment by AndrewGable** [comment]
Sorry hit the wrong button haha

File: README.md (hunk 1 above)
Comment (AndrewGable) [reply README.md:3]: From my tests I couldn't use a local path, but this is somewhat a relative path as it should be using whatever the favicon from master is.

**Review by marcaaron — APPROVED** [review]
🔥

**Pull request merged**